client/components/jetpack/connection-health/index.tsx

```diff
@@ -42,27 +33,27 @@ export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 		return;
 	}
 
-	return (
-		<>
-			<TrackComponentView
-				eventName="calypso_jetpack_connection_health_issue_view"
-				eventProperties={ { type: 'default' } }
+	if ( jetpackConnectionHealth?.error === DNS_ERROR ) {
+		return (
+			<ErrorNotice
+				eventType="dns"
+				errorText={ translate(
+					"Your domain is not properly set up to point to your site. Reset your domain's A records in the Domains section to fix this."
```

Comment:
This message is confusing to me...

Reply:
I stole it from the other component. Do you think we should better use one from design?

> Your primary domain is pointing to an unavailable site. Please edit your domain's DNS records to point them to this site.

Reply:
Hm, not sure I love that either. I'd go with something like this for now:

```suggestion
					"Jetpack is unable to connect to your domain. Please update your domain's DNS records so they're pointed properly to your site."
```